tsconfig.base.v8.json

```diff
@@ -0,0 +1,61 @@
+    "baseUrl": ".",
+    "paths": {
+      "@fluentui/api-docs": ["packages/api-docs/src/index.ts"],
+      "@fluentui/azure-themes": ["packages/azure-themes/src/index.ts"],
+      "@fluentui/codemods": ["packages/codemods/src/index.ts"],
+      "@fluentui/date-time-utilities": ["packages/date-time-utilities/src/index.ts"],
+      "@fluentui/example-data": ["packages/example-data/src/index.ts"],
+      "@fluentui/font-icons-mdl2": ["packages/font-icons-mdl2/src/index.ts"],
+      "@fluentui/foundation-legacy": ["packages/foundation-legacy/src/index.ts"],
+      "@fluentui/jest-serializer-merge-styles": ["packages/jest-serializer-merge-styles/src/index.ts"],
+      "@fluentui/merge-styles": ["packages/merge-styles/src/index.ts"],
+      "@fluentui/react": ["packages/react/src/index.ts"],
+      "@fluentui/react/lib/*": ["packages/react/src/*"],
+      "@fluentui/react-date-time": ["packages/react-date-time/src/index.ts"],
+      "@fluentui/react-experiments": ["packages/react-experiments/src/index.ts"],
+      "@fluentui/react-experiments/lib/*": ["packages/react-experiments/src/*"],
+      "@fluentui/react-file-type-icons": ["packages/react-file-type-icons/src/index.ts"],
+      "@fluentui/react-focus": ["packages/react-focus/src/index.ts"],
+      "@fluentui/react-hooks": ["packages/react-hooks/src/index.ts"],
+      "@fluentui/scheme-utilities": ["packages/scheme-utilities/src/index.ts"],
+      "@fluentui/set-version": ["packages/set-version/src/index.ts"],
+      "@fluentui/style-utilities": ["packages/style-utilities/src/index.ts"],
+      "@fluentui/test-utilities": ["packages/test-utilities/src/index.ts"],
+      "@fluentui/theme-samples": ["packages/theme-samples/src/index.ts"],
+      "@fluentui/utilities": ["packages/utilities/src/index.ts"],
+      "@fluentui/webpack-utilities": ["packages/utilities/src/index.ts"],
+      "@fluentui/dom-utilities": ["packages/dom-utilities/src/index.ts"],
+      "@fluentui/theme": ["packages/theme/src/index.ts"],
+      "@fluentui/react-cards": ["packages/react-cards/src/index.ts"],
+      "@fluentui/react-charting": ["packages/react-charting/src/index.ts"],
+      "@fluentui/react-window-provider": ["packages/react-window-provider/src/index.ts"],
+      "@fluentui/react-icons-mdl2": ["packages/react-icons-mdl2/src/index.ts"],
+      "@fluentui/react-icons-mdl2-branded": ["packages/react-icons-mdl2-branded/src/index.ts"],
+      "@fluentui/react-icon-provider": ["packages/react-icon-provider/src/index.ts"],
+      "@fluentui/react-examples": ["packages/react-examples/src/index.ts"],
+      "@fluentui/react-examples/lib/*": ["packages/react-examples/src/*"],
+      "@fluentui/public-docsite-setup": ["packages/public-docsite-setup/src/index.ts"],
+      "@fluentui/react-docsite-components": ["packages/react-docsite-components/src/index.ts"],
+      "@fluentui/react-docsite-components/lib/index2": ["packages/react-docsite-components/src/index2.ts"],
+      "@fluentui/monaco-editor": [
```

Comment:
this is quite tricky as monaco editor definitions are created on build, for sake of intellisense this is all that's needed, but if it would be used for build the `monaco-editor` build would need to be trigger in advance. thankfully only public-docsite v8 uses this editor in whole monorepo so everything else is all set.